css/css-images/gradient/gradient-eval-001.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html>
+ <head>
+  <title>Gradient interpolation</title>
+  <meta name="fuzzy" content="maxDifference=2"/>
+  <link rel="help" href="https://www.w3.org/TR/css-color-4/#interpolation">
+  <meta name="assert" content="A narrow segment of a wide gradient is essentially a single color, allowing for anti-aliasing"/>
+  <link rel="match" href="../../reference/ref-filled-green-100px-square-only.html"/>
```

Comment:
That should be relocated here, and renamed to whatever-ref.html to keep test and ref adjacent